quickwit/quickwit-config/src/node_config/serialize.rs

```diff
@@ -308,6 +308,15 @@ fn validate(node_config: &NodeConfig) -> anyhow::Result<()> {
     if node_config.peer_seeds.is_empty() {
         warn!("Peer seed list is empty.");
     }
+
+    if node_config.metastore_uri.protocol().is_file() {
+        warn!(
+            "metastore_uri = {}. Using file-backed metastore comes with certain limitations, \
+             please refer to the specific documentation for details.",
+            node_config.metastore_uri
+        );
```

Comment:
```suggestion
        warn!(
            metastore_uri = %node_config.metastore_uri, "Using file-backed metastore comes with certain limitations, please refer to the specific documentation for details."
        );
```

Reply:
(also I am not sure uri implements display `Display` but you get the gist.)

Reply:
The comment is not super helpful. We should have a link to the doc or something for people to actually know what the limitations are.

Reply:
@fulmicoton 
Thanks for reviewing
I mean to give the link as https://quickwit.io/docs/configuration/metastore-config
but I am not sure if the link will be persistent